Sources/MintKit/Mint.swift

```diff
@@ -268,6 +268,23 @@ public class Mint {
             }
         }
 
+        // copy manpages
+        do {
```

Comment:
Surrounding with `do` statement because I didn't want to spill out `allFiles` variable to other scope.

Reply:
Not sure where you suggest that `func` to put? I leave it for you to refactor later. 👌
It's a good practice to surround with `do` to improve readability.

Reply:
Fixed in 21f5b87 .